### PR TITLE
Fix course setup and batch enrolling scripts

### PR DESCRIPTION
**Pull request by idegtiarov**

### CODE REVIEW

File: data/BKT/problist.tsv [hunk 1]
```diff
@@ -1,84 +1,84 @@
-axis_0	x axis
```
Comment (idegtiarov) [inline data/BKT/problist.tsv]: done

File: config.py [hunk 2]
```diff
@@ -5,4 +5,5 @@
 EDXADAPT = {
     'HOST': '52.210.53.36',
     'PORT': '8080',
+    'COURSE_ID': 'course-v1:CMU+STAT101+2014_T1'
```
Comment (idegtiarov) [inline config.py]: done

File: tools/course_setup.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+"""
+Script for manual course setup in edx-adapt application.
+
+Course's required data contains in two csv files skills.csv and problems.csv.
```
Comment (idegtiarov) [inline tools/course_setup.py]: done

File: tools/course_setup.py [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+DO_BASELINE_SETUP = False
+COURSE_ID = EDX['COURSE_ID']
+
+SKILL2INDEX = {}  # Skill names to their indices
+PROBLEM2SKILL = {}  # Dictionary of problems (seq_problemid) to their skill indices
+TEST2SKILL = {}  # Dictionary of pretest problems to their skill indices
+NUM_PRETEST_PER_SKILL = []  # Number of pretest questions per skill
+SKILL2SEQ = []  # list of sets, where the list index is the skill index, and the value is the set of sequentials with
+# problems corresponding to that skill
+# The above are filled using skills.csv file, so any changes to the skill structure must be specified in the skills.csv
```
Comment (idegtiarov) [inline tools/course_setup.py]: done

File: tools/course_setup.py [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+        type=str,
+        help='path to dir with course setup required csv files: skills.csv and problems.csv.')
+
+    parser.add_argument(
+        '--host',
+        dest='host',
+        type=str,
+        default=EDXADAPT['HOST'] if not required else None,
+        required=required,
+        help='host of the edx-adapt server.'
```
Comment (idegtiarov) [inline tools/course_setup.py]: done

File: tools/course_setup.py [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+    )
+    parser.add_argument(
+        '--host_edx',
+        dest='host_edx',
+        type=str,
+        default=EDX['HOST'] if not required else None,
+        required=required,
+        help='host of the edx platform server.'
```
Comment (idegtiarov) [inline tools/course_setup.py]: done

File: tools/course_setup.py [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+    )
+    parser.add_argument(
+        '--port',
+        dest='port',
+        type=int,
+        default=EDXADAPT['PORT'] if not required else None,
+        required=required,
+        help='port of the edx-adapt server.'
```
Comment (idegtiarov) [inline tools/course_setup.py]: done

File: tools/course_setup.py [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+        type=str,
+        default=None,
+        help=(
+            "name of section with adaptive problems in Edx course which is added into problem's url. Not need if "
+            "coure_id contains section otherwise mandatory"
+        )
+    )
+    parser.add_argument(
+        '--https',
+        help="Protocol tutor urls are started with"
```
Comment (idegtiarov) [inline tools/course_setup.py]: improve parameter description

File: edx_adapt/tests/logic_tests.py [hunk 9]
```diff
@@ -101,7 +101,7 @@ def test_course_has_skills(self):
     def test_course_problem_fulfilled(self):
         problems = json.loads(self.app.get(base_api_path + '/{}'.format(self.course_id)).data)
         self.assertTrue(problems, msg='Not any problem is found in course')
-        # NOTE(idegtiarov) Assert correct value is based on amount of problems defined in problist.tsv file, if this
+        # NOTE(idegtiarov) Assert correct value is based on amount of problems defined in problem.csv file, if this
```
Comment (z4y4ts) [inline edx_adapt/tests/logic_tests.py]: problems.csv

**Pull request merged**